examples/timeSeries-hand-gestures/index.html

```diff
@@ -0,0 +1,22 @@
+<!--
+  👋 Hello! This is an ml5.js example made and shared with ❤️.
+  Learn more about the ml5.js project: https://ml5js.org/
+  ml5.js license and Code of Conduct: https://github.com/ml5js/ml5-next-gen/blob/main/LICENSE.md
+ 
+  This example demonstrates training a Sign Language classifier through ml5.TimeSeries.
```

Comment:
Hi professor, I agree, it might be too specific for casual users, thus I changed it into hand gesture classifier, at the end the training example you can choose whatever word you want, so its not gonna be an issue, however for the load model, its hard to think of a moving gesture that everyone knows (thumbs up comes to mind but movements are minimal) so I still continued with the ASL goodbye and hello and included some links so users can get to know how the actual gesture is done

Reply:
This seems like a very sensible compromise!